elements/lisk-p2p/src/peer_pool.ts

```diff
@@ -490,16 +496,75 @@ export class PeerPool extends EventEmitter {
 		})();
 	}
 
+	private _selectPeersForEviction(peers: Peer[]): Peer[] {
+		const PEER_PROTECTION_PERCENTAGE = 0.08;
```

Comment:
I think there is a naming issue here. It's difficult to understand how many peers of each kind are being protected.